### ISSUE TITLE
Missing declaration file (image-lazy-load.service.d.ts)

### ISSUE THREAD

**Issue by fetchandadd** [NathanWalker/ng2-image-lazy-load#13]
After I installed ng2-image-lazy-load my typescript compilation failed:

```
[10:20:16] Starting 'compile:ts'...
[...]/node_modules/ng2-image-lazy-load/index.d.ts(2,15): error TS2307: Cannot find module './src/app/services/image-lazy-load.service'. undefined
[...]/node_modules/ng2-image-lazy-load/src/app/directives/image-lazy-load.directive.d.ts(4,62): error TS2307: Cannot find module '../services/image-lazy-load.service'. undefined
[10:20:20] TypeScript: 2 semantic errors
```

After a deeper look insight the 'src/app/services'-folder of ng2-image-lazy-load I noticed that there isn't a image-lazy-load.service.d.ts-file. 

I'm not sure if this is a mistake on my part or the missing declaration file is the reason. But after writing and adding the image-lazy-load.service.d.ts-file by myself locally the compilation succeeded.

npm --version: 3.7.3
node --version: 5.9.1

tsconfig.json

```
  "compilerOptions": {
    "target": "es5",
    "module": "commonjs",
    "moduleResolution": "node",
    "sourceMap": true,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "removeComments": false,
    "noImplicitAny": true,
    "suppressImplicitAnyIndexErrors": true,
    "outDir": "public/dist"
  },
```

**Comment by jinder** [comment]
Also seeing this issue with the NPM package.

**Comment by ksprwhite** [comment]
Same issue

**Comment by stefanfr** [comment]
Same here

**Comment by digitalkaoz** [comment]
@timtegeler would you mind to provide a PR then?

**Comment by fetchandadd** [comment]
ok i will provide a PR tomorrow

**Comment by NathanWalker** [comment]
A PR would be appreciated.
I won't get a chance to fix this til Monday.

**Comment by digitalkaoz** [comment]
See the linked PR

**Comment by fetchandadd** [comment]
If further contribution is necessary let me know

**Comment by NathanWalker** [comment]
Published with `2.0.8` now.
